blaze/compatibility.py

```diff
@@ -66,6 +80,7 @@ def boundmethod(func, instance):
     boundmethod = MethodType
 
     from itertools import zip_longest
+    from collections import abc as collections_abc
```

Comment:
I didn't realize this. Should I just import them from collections at the use site?